handlers/easy_info/__init__.py

```diff
@@ -1,10 +1,40 @@
 from eventhandler import EventHandler
 
+ASSIGN_MSG = '@highfive: assign me'
```

Comment:
The name in this message should probably be derived from the account name in the configuration.